widgets/nuxeo-operation-button.html

```diff
@@ -76,14 +82,78 @@
           type: Object,
           value: null,
           notify: true
+        },
+
+        /* The text or i18n key to display in the notification. */
```

Comment:
For clarity, I'd just add in this comment that this will only be displayed if no error occurred, and that specifying no `notification` whatsoever will result in not toast being shown.

Reply:
Should we have a default notification? I kept a default event and detail....